Makefile

```diff
@@ -0,0 +1,71 @@
+ANDROID_HOME=$(PWD)/../android
+export ANDROID_HOME
+
+BUILD_TOOLS_VERSION != sed -n 's/^ *- build-tools-//p' .travis.yml
+ANDROID_PLATFORM_VERSION != (sed -n 's/^ *- yes | sdkmanager "platforms;android-\([0-9]*\)"/\1/p' .travis.yml; echo 26) | head -n1
+
+GIT_DISCOVERY_ACROSS_FILESYSTEM=y
+export GIT_DISCOVERY_ACROSS_FILESYSTEM
+
+SDKS = "platforms;android-$(ANDROID_PLATFORM_VERSION)" "build-tools;$(BUILD_TOOLS_VERSION)" "extras;google;m2repository"
+ANDROID_CLI_TOOLS_ZIP=commandlinetools-linux-7302050_latest.zip
```

Comment:
I'm still not very happy with the fixed version in the `commandlinetools`. As far as I know, they can be simply installed using the OS common package managers like `sudo apt install android-tools` (see [How To Set Up Android Development Tools On Linux](https://www.addictivetips.com/ubuntu-linux-tips/set-up-android-development-tools-on-linux)).

Hence, why not their installation status?